api/HTMLHeadElement.json

```diff
@@ -61,12 +61,11 @@
               "version_removed": "79"
             },
             "firefox": {
-              "version_added": true,
-              "version_removed": "7"
+              "version_added": "1",
+              "version_removed": "4"
```

Comment:
Here's Firefox 4 again. Was Firefox 4 very special, or why does it show up so much?

http://mdn-bcd-collector.appspot.com/tests/api/HTMLHeadElement/profile returns false for me in Firefox 3.6 on Windows XP (BrowserStack) as well, is something wrong here?

Reply:
Hmmm...Firefox 3.6 on Windows XP and 7 are returning `true` for me.  I know that sometimes with the older Firefox versions, BrowserStack doesn't close the previous version before attempting to open the tabs, resulting in it opening the same version (so if you're on Firefox 4, but then switch to 3.6, it doesn't close Firefox 4 and ends up still using Firefox 4).

Reply:
Oh, that sounds concerning! Is this when running under Selenium, or manually?

Reply:
Luckily, this is just when running tests manually; running in Selenium has no problem.  I've sent a message to BrowserStack support about it, hopefully the problem will be resolved soon!